RELEASE-NOTES.md

```diff
@@ -16,3 +16,4 @@
 * Fixed registry token connection issues
 * Fixes in commands belonging to `snow snowpark compute-pool` and `snow snowpark services` groups
 * Removed duplicated short option names in a few commands
```

Comment:
I think we should add list of these commands

Reply:
Done